docs/reference/argocd.md

```diff
@@ -189,9 +189,7 @@ metadata:
     example: openshift-oauth
 spec:
   dex:
-    image: quay.io/redhat-cop/dex
```

Comment:
Do you want to get rid of this in both examples and metadata. I understand that the various options for Dex are listed in the table but I feel this changes are  good to have in the argocd.md file

Reply:
The image in this repository was created in order to provide a version with the functionality that support OpenShift authentication as the most related fixes were not in `v2.22.0`. `v2.28.0` does contain these fixes

Reply:
Thanks @sabre1041